src/commands/BootstrapCommand.js

```diff
@@ -129,18 +129,20 @@ class BootstrapCommand extends Command {
       if (err) {
         callback(err);
       } else {
-        this.logger.success("", `Bootstrapped ${this.filteredPackages.length} packages`);
+        this.logger.success("", `Bootstrapped ${this.filteredPackages.length} package${this.pluralize()}`);
```

Comment:
Let's just set this as an instance property near the beginning of the `initialize()` method, no need to have a one-liner instance method.
```
const filteredLength = this.filteredPackages.length;
this.packageCountLabel = `${filteredLength} package${filteredLength > 1 's' : ''}`;
```
in `execute()`'s inner callback:
```
this.logger.success("", `Bootstrapped ${this.packageCountLabel}`);
```
in `bootstrapPackages()`:
```
this.logger.info("", `Bootstrapping ${this.packageCountLabel}`);
```